Authentication Methods Registration Campaign Graph Docs (#12112)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/authenticationmethodspolicy.md

---
title: "authenticationMethodsPolicy resource type"

> description: "Authentication methods policies."

Reviewer: ```suggestion
description: "Define authentication methods and the users that are allowed to use them to sign in and perform multi-factor authentication (MFA)."
```

author: "mmcla"
localization_priority: Normal

> ms.prod: "microsoft-identity-platform"

Reviewer: ```suggestion
ms.prod: "identity-and-sign-in"
```

doc_type: resourcePageType
---

# authenticationMethodsPolicy resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Authentication methods policies define authentication methods and the users that are allowed to use them to sign in and perform multi-factor authentication (MFA) in Azure Active Directory (Azure AD).

Reviewer: ```suggestion
Define authentication methods and the users that are allowed to use them to sign in and perform multi-factor authentication (MFA) in Azure Active Directory (Azure AD).
```

## Methods
|Method|Return type|Description|
|:---|:---|:---|
|[Get authenticationMethodsPolicy](../api/authenticationmethodspolicy-get.md)|[authenticationMethodsPolicy](../resources/authenticationmethodspolicy.md)|Read the properties and relationships of an [authenticationMethodsPolicy](../resources/authenticationmethodspolicy.md) object.|
|[Update authenticationMethodsPolicy](../api/authenticationmethodspolicy-update.md)|[authenticationMethodsPolicy](../resources/authenticationmethodspolicy.md)|Update the properties of an [authenticationMethodsPolicy](../resources/authenticationmethodspolicy.md) object.|

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |description|String|A description of the policy|

Reviewer: ```suggestion
|description|String|A description of the policy.|
```

|displayName|String|The name of the policy.|

> |id|String|The ID of the policy. Inherited from [entity](../resources/entity.md)|

Reviewer: ```suggestion
|id|String|The identifier of the policy. Inherited from [entity](../resources/entity.md)|
```

|lastModifiedDateTime|DateTimeOffset|The datetime of the last update to the policy.|
|policyVersion|String|The version of the policy in use.|

> |registrationEnforcement|[registrationEnforcement](../resources/registrationenforcement.md)|Settings for enforcing registration at sign-in time. Can currently be used to nudge users to setup targeted authentication methods.|

Reviewer: ```suggestion
|registrationEnforcement|[registrationEnforcement](../resources/registrationenforcement.md)|Settings for enforcing registration at sign-in time. This property can be used to remind users to set up targeted authentication methods.|
```

## Relationships
|Relationship|Type|Description|
|:---|:---|:---|

> |authenticationMethodConfigurations|[authenticationMethodConfiguration](../resources/authenticationmethodconfiguration.md) collection|Settings for each authentication method.|

Reviewer: ```suggestion
|authenticationMethodConfigurations|[authenticationMethodConfiguration](../resources/authenticationmethodconfiguration.md) collection|Represents the settings for each authentication method.|
```
